Adyen/Assets/Generated/LocalizationKey.swift

```diff
@@ -2,7 +2,7 @@
 // Copyright (c) 2021 Adyen N.V.
 //
 // This file is open source and available under the MIT license. See the LICENSE file for more info.
-// This file is autogenerated. Please do not modify it.
```

Comment:
Let's keep this one :D

Reply:
How is it removed? :D i did not touch it

Reply:
probably `swiftformat`